pydis_site/apps/api/viewsets/bot/pagination.py

```diff
@@ -0,0 +1,12 @@
+from rest_framework.pagination import LimitOffsetPagination
+from rest_framework.response import Response
+
+
+class LimitSetPagination(LimitOffsetPagination):
+    """Extend LimitOffsetPagination."""
```

Comment:
The documentation doesn't have to be extremely detailed, but what you currently have is too vague. It doesn't really provide any information the reader doesn't already know.